skimage/feature/texture.py

```diff
@@ -291,3 +293,130 @@ def local_binary_pattern(image, P, R, method='default'):
     image = np.ascontiguousarray(image, dtype=np.double)
     output = _local_binary_pattern(image, P, R, methods[method.lower()])
     return output
+
+def multiblock_local_binary_pattern(int_image, x, y, width, height):
+    """Multi-block local binary pattern.
+
+    MB-LBP is an extension of LBP that can be computed on many
+    scales in a constant time using integral image. It consists of
+    9 equal-sized rectangles. Sum of pixels' intensity values
```

Comment:
I can think of 3 things
1. Throw error ( I am not much in favour of this ).
2. Modify the window, and issue warning.
For `2`, we can either round off, or, adjust the window size, like `21 -> 7,7,7` , `22 -> 7,8,7`, `23->8,7,8`
@stefanv What do you think ?